Base.rte/AI/NativeHumanAI.lua

```diff
@@ -727,9 +730,17 @@ function NativeHumanAI:CreateAttackBehavior(Owner)
 	
 	local dist = SceneMan:ShortestDistance(Owner.Pos, self.Target.Pos, false).Magnitude
 
-	if self.Target and IsADoor(self.Target) and Owner:EquipNamedDevice("Heavy Digger", true) then
-		self.NextBehavior = coroutine.create(HumanBehaviors.AttackTarget)
-		self.NextBehaviorName = "AttackTarget"
+	if IsADoor(self.Target) then 
+		if Owner:EquipNamedDevice("Heavy Digger", true) then
```

Comment:
Remote Explosive doesn't have any AI functionality yet, which is why I didn't add it. Heavy Digger is favored over explosives so that the AI is less likely to kill itself lol

Reply:
Ahhh, that all makes sense haha. Cool